mantle/kola/harness.go

```diff
@@ -451,11 +453,19 @@ func ParseDenyListYaml(pltfrm string) error {
 			snoozeDate, err := time.Parse(snoozeFormat, obj.SnoozeDate)
```

Comment:
I think there are some corner cases in this logic that aren't being accounted for. The logic gets complicated with warn and snooze both in play I tried to simplify the logic (though it was hard) in a commit in https://github.com/dustymabe/coreos-assembler/commits/PR3539 - let me know what you think

Reply:
looks good. should i cherry-pick that patch?